suite/tests/CMakeLists.txt

```diff
@@ -3258,6 +3258,27 @@ endif ()
       # XXX i#1732: add more tests.  However, we don't want the suite to fail
       # b/c our tests have too-new instrs for older machines.
     endif (X86)
+
+    ###########################################################################
+    # drdisasm tests
+    # XXX: We should try to export or share the test list, filtering, matching,
+    # and other code here to support tests being declared locally rather than
+    # in one central place.
+    if (AARCH64)
+      set(drdisasm_args "f94017a0" "a9be7bfd")
+    elseif (ARM)
+      set(drdisasm_args "6813" "f243" "42c8" "b2db")
+    else ()
+      set(drdisasm_args "66" "90" "c4" "e2" "65" "90" "14" "80")
+    endif ()
+    torunonly_api(tool.drdisasm drdisasm
+      "../../clients/drdisasm/test_simple.c" "" "${drdisasm_args}" ON)
```

Comment:
I take it that this "../../clients/drdisasm/test_simple.c" is just a dummy name so that the test compares with the template file right?

Reply:
Yes.  Looking at the code: any extension will do, so I will s/.c/.template/ which should then be clearer.